changelogs/fragments/300-set-ip-dhcp-option-force-default.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - api_info, api_modify - set default for ``force`` in ``ip dhcp-server option`` to an explicit false (https://github.com/ansible-collections/community.routeros/issues/300).
```

BTW, you can also put both fragments into the same YML file, as two list entries in `minor_changes`.

```suggestion
  - api_info, api_modify - set default for ``force`` in ``ip dhcp-server option`` to an explicit ``false`` (https://github.com/ansible-collections/community.routeros/issues/300).
```

---

Done. I deliberately used separate files because they're added by separate commits.

---

Fine for me. But please note that the commits will get squashed on merge anyway :)